recipes/treelib/LICENSE

```diff
@@ -0,0 +1,21 @@
+treelib - Simple to use for you.
```

Comment:
This LICENSE file can be removed. It is included in the pypi tarball.

Reply:
Done, thanks